store/server/features/okrplanning/okr/keyresult/queries.ts

```
@@ -1,36 +1,27 @@
 import { useAuthenticationStore } from '@/store/uistate/features/authentication';
 import { KeyResult } from '@/store/uistate/features/okrplanning/okr/interface';
 import { OKR_AND_PLANNING_URL } from '@/utils/constants';
-import axios from 'axios';
+import { crudRequest } from '@/utils/crudRequest';
+import { requestHeader } from '@/helpers/requestHeader';
 import { useQuery } from 'react-query';
+
 const token = useAuthenticationStore.getState().token;
```

ESLint

[error] 'token' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)